docusaurus/docs/recipes/contentful.md

```diff
@@ -0,0 +1,513 @@
+---
+id: contentful
+title: Implementing Contentful
+sidebar_label: Contentful
+---
+
+This recipe aims to guide you through the implementation of Contetful in the project and also provides custom solutions for different use-cases.  
```

Comment:
```suggestion
This recipe aims to guide you through the implementation of Contentful in the project and also provide custom solutions for different use-cases.  
```

Reply:
Misspell fixed, but I think `provides` is more correct